packages/canon/src/components/Heading/Heading.tsx

```diff
@@ -14,10 +14,13 @@
  * limitations under the License.
  */
 
+'use client';
```

Comment:
Backstage doesn't do any server-side rendering, so this directive doesn't make sense when this component is used in production. It seems a little counter-productive to have changes in components that are motivated strictly by the Canon website - perhaps we can move this directive further up the tree into website code to get around this need?

Reply:
I could but I think this is good practice anyway. All Base UI components have this directive as well. The more we can make this library React 19 ready the better. https://github.com/mui/base-ui/blob/master/packages/react/src/accordion/root/AccordionRoot.tsx

Reply:
I removed all 'use client' directives. I'll do that as a separate PR.